vividus-to-zephyr-exporter/build.gradle

```diff
@@ -12,9 +12,19 @@ apply plugin: org.springframework.boot.gradle.plugin.SpringBootPlugin
 
 project.description = 'Vividus to Zephyr exporter'
 
+configurations.testImplementation {
+    exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j18-impl'
+    exclude group: 'ch.qos.logback', module: 'logback-classic'
+}
+
+configurations.all {
+    exclude group: 'org.apache.logging.log4j', module: 'log4j-to-slf4j'
+}
```

Comment:
it seems this logic is common for all exporter modules, thus it should be moved to a separate gradle script and then re-used